1-Introduction/1-intro-to-ML/translations/README.fr.md

```diff
@@ -0,0 +1,109 @@
+# Introduction au machine learning
+
+[![ML, AI, deep learning - Quelle est la différence ?](https://img.youtube.com/vi/lTd9RSxS9ZE/0.jpg)](https://youtu.be/lTd9RSxS9ZE "ML, AI, deep learning - What's the difference?")
+
+> 🎥 Cliquer sur l'image ci-dessus afin de regarder une vidéo expliquant la différence entre machine learning, AI et deep learning.
+
+## [Quizz de pré-conférence](https://jolly-sea-0a877260f.azurestaticapps.net/quiz/1/)
```

Comment:
Let's use the spelling 'quiz' and I think our other translator is using 'pré-lecture' - what do you think?

Reply:
This was one of the difficult point to translate actually.
Keeping lecture, in french, would actually change the original meaning since "lecture", in French, can be translated to reading, in English. Moreover, the actual meaning for "prélecture" (without hyphen) is the proofreading of test books before sending them back to their author (https://www.cnrtl.fr/definition/pr%C3%A9lecture https://www.lalanguefrancaise.com/dictionnaire/definition/prelecture)
If I'm not wrong, lecture here is for lesson, therefore I've used "conférence" which is the equivalent of a university lesson/course. For example, lecturers are usually translated as "conférenciers".
I actually do not like the use of "pré-conférence" either, it might be better to use some words with broader meaning like "préalables". It would be great if @ManuSquall could give some feedback about it, so we can avoid some Anglicism.

For quiz, it is misspelled, I need to change it, thanks for pointing it out.

Reply:
This was one of the difficult point to translate actually.
Keeping lecture, in french, would actually change the original meaning since "lecture", in French, can be translated to reading, in English. Moreover, the actual meaning for "prélecture" (without hyphen) is the proofreading of test books before sending them back to their author (https://www.cnrtl.fr/definition/pr%C3%A9lecture https://www.lalanguefrancaise.com/dictionnaire/definition/prelecture)
If I'm not wrong, lecture here is for lesson, therefore I've used "conférence" which is the equivalent of a university lesson/course. For example, lecturers are usually translated as "conférenciers".
I actually do not like the use of "pré-conférence" either, it might be better to use some words with broader meaning like "préalables". It would be great if @ManuSquall could give some feedback about it, so we can avoid some Anglicism.

For quiz, it is misspelled, I need to change it, thanks for pointing it out.

Reply:
You're right about the reading meaning @simplg let's take préconférence and postconférence. But still hesitating to add or not the hyphen 👀